src/v3/src/components/Button/Button.tsx

```diff
@@ -73,6 +77,8 @@ const Button: UISchemaElementComponent<{
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...(ariaLabel && { 'aria-label': ariaLabel } )}
     >
+      {loading && <CircularProgress />}
+      {loading && ' '}
```

Comment:
I think you can use MUI's loading button component instead, see: https://mui.com/material-ui/react-button/#loading-button

Reply:
The loading button is in the @mui/lab package which looks like an experimental package I think?  Do we want to add that package?

Reply:
Oh I didn't notice that, yeah since this is in the lab and since the lab can introduce breaking changes, let's not use that (unless someone else feels differently). 

Would it be possible to update the css for the progress indicator that is added to the button? I was thinking making it aligned similar to MUI's loading button, or if that is too much, can the button text be removed and just display the spinner? 

Additionally, if it is simple enough, can the color of the spinner in the button be changed to match MUI's loading button? 

This is just so it looks a little better visually. 

<img width="510" alt="image" src="https://user-images.githubusercontent.com/97472729/191557122-772aa3d4-99ff-4615-b806-766388172d45.png">